RetailCoder.VBE/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.UI;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Refactorings.MoveCloserToUsage
+{
+    public class MoveCloserToUsageRefactoring : IRefactoring
+    {
+        private readonly List<Declaration> _declarations;
+        private readonly IActiveCodePaneEditor _editor;
+        private readonly IMessageBox _messageBox;
```

Comment:
Not sure why the refactoring needs to know about that message box... Seems like you're doing more than just the refactoring in here.